### PR TITLE
Update live-deploy.yml

### PR DESCRIPTION
**Pull request by sjmiles**
Bump to `set-node@v2-beta` so minor version updates are automatic.

### CODE REVIEW

File: .github/workflows/live-deploy.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ jobs:
       - uses: actions/checkout@v2
 
       - name: Setup Node
-        uses: actions/setup-node@v2.1.0
```
Comment (alxrsngrtn) [inline .github/workflows/live-deploy.yml:16]: I don't think `2.3.0` exists. See the releases page: https://github.com/actions/setup-node/releases

Furthermore, check out the comparison from the latest semantic release and `v2-beta`: https://github.com/actions/setup-node/compare/v2-beta...v2.1.3

It looks like we can use `v2-beta` to stay on the latest v2 changes w/o having to worry about minor versions.

**Review by alxrsngrtn — CHANGES_REQUESTED** [review]

File: .github/workflows/live-deploy.yml (hunk 1 above)
Comment (sjmiles) [reply .github/workflows/live-deploy.yml:16]: Ooops, I meant 2.1.3

Comment (sjmiles) [reply .github/workflows/live-deploy.yml:16]: Ok

**Comment by sjmiles** [comment]
Thank you for the save. :wav:

**Review by alxrsngrtn — APPROVED** [review]
Looks good!

**Pull request merged**